.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build
```

Comment:
Is there a way to run this locally before creating the pull request?

Maybe we can do some research about [this](https://www.reddit.com/r/vscode/comments/1gyds8l/run_your_github_actions_locally_in_vs_code/) and we can add a section to the README to recommend using this to trigger actions locally without needing to open a PR.